src/main/java/org/gradle/profiler/mutations/CopyFileMutator.java

```diff
@@ -0,0 +1,56 @@
+package org.gradle.profiler.mutations;
+
+import com.google.common.base.Strings;
+import com.google.common.io.Files;
+import com.typesafe.config.Config;
+import org.gradle.profiler.ConfigUtil;
+import org.gradle.profiler.ScenarioContext;
+
+import java.io.File;
+import java.io.IOException;
+
+public class CopyFileMutator extends AbstractFileSystemMutator {
+
+    private final File source;
+    private final File target;
+
+    public CopyFileMutator(File source, File target) {
+        this.source = source;
+        this.target = target;
+    }
+
+    @SuppressWarnings("UnstableApiUsage")
```

Comment:
❔ Which API is unstable, maybe there is a stable replacement?

Reply:
It's Guava. I think upgrading Guava is the right solution here. Let me take a look.